### PR TITLE
basic card functionality

### PR DESCRIPTION
**Pull request by jasontduong**
-displays image to maximum width, scales down if too big; vice versa if too small. 
-card height changes to match image
-card top-bar displayed with placeholders for profile-icon, username, settings drop-down, social media icon
-flex box adjusts correctly when adding text

### CODE REVIEW

File: app/src/App.css [hunk 1]
```diff
@@ -1,3 +1,78 @@
+.dashboard {
```
Comment (ahn6) [inline app/src/App.css:1]: Can you move this CSS into its own file? Ideally, we will use `App.css` as a top level CSS file

something like `/app/assets/stylesheets/dashboard-card.css` and `/app/assets/stylesheets/card-bar.css`

**Comment by ahn6** [comment]
👍

**Comment by KennethDamCS** [comment]
:thumbsup:

File: app/src/App.css (hunk 1 above)
Comment (jasontduong) [reply app/src/App.css:1]: surely!

**Pull request merged**